src/main/java/org/generationcp/middleware/service/impl/study/StudyServiceImpl.java

```diff
@@ -520,6 +551,35 @@ public StudyDetailsDto getStudyDetailsForGeolocation(final Integer geolocationId
 		}
 	}
 
+
+	private List<MeasurementVariable> createGeolocationVariables(final List<MeasurementVariable> measurementVariables, final Integer geolocationId) {
```

Comment:
This is fine for now. This will eventually be unnecessary once we tackle nd_geolocation re-engineering